src/main/java/commands/PreviewCommand.java

```diff
@@ -0,0 +1,67 @@
+package commands;
+
+import access.Access;
+import manager.chapter.DueChapter;
+import scheduler.Scheduler;
+import storage.Storage;
+import ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import access.Access;
+
+import manager.chapter.CardList;
+import manager.chapter.DueChapter;
+
+import scheduler.Scheduler;
+import storage.Storage;
+import ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class PreviewCommand extends Command {
+    public static final String COMMAND_WORD = "preview";
+    public ArrayList<DueChapter> allChapters;
+    public ArrayList<DueChapter> dueChapters;
+
+    private void loadAllChapters(Storage storage, Ui ui) {
+        try {
+            allChapters = storage.loadAllDueChapters(ui);
+        } catch (FileNotFoundException e) {
+            ui.showToUser(Ui.UNABLE_TO_LOAD_EMPTY_DATABASE);

```

Comment:
maybe you can do throw new FileNotFoundException(message) then it will show the error at Kaji.java